### DISCUSSION TITLE
How to set the cursor to a custom component?

### DISCUSSION THREAD

**Discussion by EddieChan1993** [fyne-io/fyne#3803] [Q&A] [answered]
How to set the cursor to a custom component?

**Comment by andydotxyz** [comment]
You can't change what the cursor in an Entry field looks like. What would be the use-case for this?

**Reply by EddieChan1993** [reply]
我并不是想改外观，而是想主动将光标设定在某个组件上，比如，我在input输入文字后，我想让光标落在多文本框里面

**Reply by andydotxyz** [reply]
Google Translate:

I don't want to change the appearance, but I want to actively set the cursor on a certain component. For example, after I input text, I want the cursor to fall in the multi-text box

**Comment by andydotxyz** [comment, marked as answer]
I think you may be asking "how can I focus a different component".

In which case the answer is to use `Window.Canvas().Focus(obj)`.

**Reply by EddieChan1993** [reply]
3ks
